Dockerfile

```diff
@@ -2,8 +2,6 @@ FROM alpine
 
 RUN apk add --no-cache curl jq
 
-RUN curl -LJO https://fly.io/install.sh && chmod +x install.sh && FLYCTL_INSTALL=/usr/local ./install.sh v0.0.521
```

Comment:
We only looked for about 10-15 minutes and didn't find a way to pass things into the docker build command when loading a GitHub action. So we moved the flyctl installation into the entrypoint... a bit of extra overhead when we have the docker image cached but still run the installation every time, but no too bad